.github/workflows/nightly-build.yaml

```diff
@@ -52,3 +52,8 @@ jobs:
             > [!warning]
             > The Nightly build is for adventurous folks. It's updated daily with less-tested features and improvements.
           prerelease: true
+
+      - name: Ensure release is published
+        run: gh release edit nightly --draft=false
```

Comment:
Is there going to be an issue if the release is not a draft?

Reply:
No, as I mentioned in PR description:
> This step won't fail if re-run on published release.

Reply:
My bad. Totally missed that ...

Reply:
No worries :)